examples/polymer/vulcanize.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# vulcanize elements.html \
+#     --inline-script --inline-css --strip-comments | \
+#     crisper -h elements.build.html -j elements.build.js
+
+polybuild --maximum-crush elements/elements.html
```

Comment:
> I think one of the goals of this update was to introduce Polymer performance patterns which require a build step in this case.

Same can be said about the other frameworks build systems, which we never allowed. I do a agree that the apps should present best practices but TodoMVC isn't around performance. It's way more important that people can read the code easily and contribute back without having to setup too much. The minified elements.html for example, is against what we decided in https://github.com/tastejs/todomvc/issues/378.